integration

```diff
@@ -833,5 +833,6 @@
   "ZacheryThomas/homeassistant-smartrent",
   "zachowj/hass-node-red",
   "zigul/HomeAssistant-CEZdistribuce",
-  "ZsBT/hass-w1000-portal"
-]
\ No newline at end of file
+  "ZsBT/hass-w1000-portal",
+  "sHedC/homeassistant-mastertherm"
```

Comment:
The list is case-sensitive.

Reply:
Hi,
Thanks for the review, I have looked at this but don't know what to modify as this matches my repository that I can see.  Is this what you meant?  Or did I miss something else.

![image](https://user-images.githubusercontent.com/69856552/221358682-3d01d1bb-6a0f-4722-a923-5e9b4e0fedaa.png)

Thanks

Reply:
Alphabetially*